src/pages/index.tsx

```diff
@@ -53,11 +53,67 @@ export async function getStaticProps(
   }
 }
 
+const sections = [
+  {
+    title: 'Indonesia Starterpack',
+    description: <p>Situs atau service ini terpopuler dengan orang Indonesia</p>,
+    sites: idnStarterPack
+  },
+  {
+    title: 'Essential Developer Toolkit',
+    description: <p>Service yang ✨ anak bangsa ✨ mungkin akan pakai saat membuat app</p>,
+    sites: devStarterPack
+  },
+  {
+    title: 'Top websites USA',
+    description: (
+      <>
+        <p>
+          2022 Top websites in the USA. Sumber data website diambil dari{' '}
+          <a
+            href='https://www.semrush.com/blog/most-visited-websites/'
+            className='text-blue-700 underline'
+            target='_blank'
+            rel='noreferrer'
+          >
+            semrush.com
+          </a>
+        </p>
+        <p>
+          <em>Catatan: Ada beberapa website ditiadakan karena tidak berhubungan</em>
+        </p>
+      </>
+    ),
+    sites: websiteListUSA
+  },
+  {
+    title: 'Linux Starterpack',
+    description: <p>Service yang sering dipakai Linux user 🐧</p>,
+    sites: linuxStarterPack
+  }
+]
+
```

Comment:
Do you think that we should move this to `_data`? I think it will look a lot more cleaner.

Reply:
Yep, will move that to _data, thinking of creating a separate branch for the refactor first before reimplementing TrustPositif checking on getStaticProps

Reply:
Moved this to a separate file under _data called sections.tsx, wdyt?

Reply:
LGTM